### DISCUSSION TITLE
i can't start the game in multiplayer and in single

### DISCUSSION THREAD

**Discussion by Curse24** [Papela/Nitrox-Cracked-Mod#23] [Q&A] [answered]
when i click on play it give me an error :
System.InvalidOperationException: La séquence ne contient aucun élément correspondant à System.Linq,Enumerable.First[TSource](Enumerable"1 source, Func'2 predicate) à NitroxLauncher.Models.Patching.NitroxEntryPatch.Remove( dans NitroxLauncher\Models\Patching\NitroxEntryPatch.cs:ligne 102 a NitroxLauncher.LauncherLogic. < StartMultiplayerAsync> d_28. MoveNext0 dans NitroxLauncher\LauncherLogic.cs:ligne 229 -- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée CU- System.Runtime.ExceptionServices.ExceptionDispatchInfo.Thro w0 a System.Runtime.CompilerServices.TaskAwaiter.HandleNonSucc essAndDebuggerNotification(Task task) a NitroxLauncher.Pages.LaunchGamePage.<MultiplayerButton_C lick> d__8.MoveNext0 dans NitroxLauncher\Pages\LaunchGamePage.xaml.cs:ligne 49
![Capture d'écran 2024-11-03 132946](https://github.com/user-attachments/assets/29c03642-b425-4def-bc60-1c2be4fa6d30)

**Comment by Curse24** [comment]
im using nitrox v 1.8.0.2 and subnautica v71744

**Comment by Papela** [comment, marked as answer]
I recommend using: **mar-2023 71288**
On very new or old versions the mod does not work properly.
